config/setup/yurt-tunnel-server.yaml

```diff
@@ -127,6 +127,7 @@ spec:
         - yurt-tunnel-server
         args:
         - --bind-address=$(NODE_IP)
+        - --insecure-bind-address=$(NODE_IP)
```

Comment:
Ensure that the tunnel-server listening address can be accessed by other cloud nodes in the cluster.

Reply:
yeah, tunnel-server need to listen on node ip for insecure requests.